packages/fxa-auth-server/docs/metrics-events.md

```diff
@@ -439,7 +441,6 @@ Possible values for `${template}` include
 | `verifyEmail`              | Email sent to verify a user's account.                          |
 | `verifyLoginEmail`         | Sign-in confirmation email was sent.                            |
 | `postVerifySecondaryEmail` | Email sent when a user has added a secondary email.             |
```

Comment:
Our precommit prettier hook was the culprit for the broken formatting. I couldn't remember what to pass in to disable it, so I temporarily removed `*.md` from `lint-staged` in `package.json` in order to get around messing it all up. 😬